data/json/hobbies.json

```
@@ -91,5 +91,101 @@
     "description": "You enjoyed camping in the vast wilderness as a means of escaping the pace of modern life.  The wilderness isn't filled with people, so it must be safe.  Right?  It must be.",
     "points": 2,
     "skills": [ { "level": 2, "name": "survival" }, { "level": 2, "name": "firstaid" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "pottery",
+    "name": "Pottery",
+    "description": "You're a master of throwing on the potter's wheel, able to shape clay into whatever shape you need.",
+    "points": 2,
+    "proficiencies": [ "prof_pottery", "prof_pottery_glazing" ],
+    "skills": [ { "level": 2, "name": "fabrication" }, { "level": 1, "name": "survival" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "handloading",
+    "name": "Handloading",
+    "description": "You learned the cheap way to shoot your ammo.  The problem is you shoot even more now.",
+    "points": 2,
+    "proficiencies": [ "prof_handloading" ],
+    "skills": [ { "level": 3, "name": "fabrication" }, { "level": 2, "name": "gun" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "game_mastering",
+    "name": "Game Master",
+    "description": "You've guided players' journeys through your dungeons and fortresses.  Now, you're last member of your party.",
+    "points": 2,
+    "traits": [ "PROF_DICMASTER" ],
```

```suggestion
    "traits": [ "PROF_DICEMASTER" ],
```
Pointed out by anothersimulacrum on Discord.